src/main/java/controller/BattleController.java

```diff
@@ -46,14 +48,16 @@ public BattleController(
             final UserOnlineService userOnlineService,
             final BattleService battleService,
             final UsWaitBattService usWaitBattService,
-            final UserService userService
+            final UserService userService,
+            final EndOfBattleService endOfBattleService
```

Comment:
Do not create EndOfBattleService, do all operations with the battle in BattleService

Reply:
done